content/desktop/release-notes.md

```diff
@@ -21,6 +21,60 @@ Docker Desktop versions older than 6 months from the latest release are not avai
 
 Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/projects/1) to see what's coming next.
 
+## 4.33.0
+
+{{< release-date date="2024-07-25" >}}
+
+{{< desktop-install-v2 all=true beta_win_arm=true version="4.33.0" build_path="/TBD/" >}}
+
+### New
+
+- Docker Debug in now generally available.
+- BuildKit now evaluates Dockerfile rules to inform you of potential issues.
+- **Resource Allocation** settings can now be accessed directly from the resource usage data displayed in the Dashboard footer.
+
+### Upgrades
+
+- [Docker Compose v2.29.1-desktop.1](https://github.com/docker/compose/releases/tag/v2.29.1-desktop.1)
+- [Docker Engine v27.1.1](https://docs.docker.com/engine/release-notes/27.1/#2711)
+- [containerd v1.7.19](https://github.com/containerd/containerd/releases/tag/v1.7.19)
+- [NVIDIA Container Toolkit v1.16.0](https://github.com/NVIDIA/nvidia-container-toolkit/releases/tag/v1.16.0)
+- [Docker Scout CLI v1.11.0](https://github.com/docker/scout-cli/releases/tag/v1.11.0)
+- [Kubernetes v1.30.2](https://github.com/kubernetes/kubernetes/releases/tag/v1.30.2)
+- Linux kernel `v6.10`
+
+### Bug fixes and enhancements
+
+#### For all platforms
+
+- Fixed an issue that caused containers started with `--net=host` and listening on an IPv6 address to be accessible from the host.
+- Improved the UX for enabling the containerd image store in the **Settings** tab.
+- Fixed an issue that caused a deadlock seen while using the `grpcfuse` filesharing option under heavy load.
+- Fixed a bug where Mac-specific admin settings were impacting other platforms.
+- The Kubernetes client now uses KIND.
```

Comment:
Please remove this line, as Kind is still only accessible via Nightly Builds, and we're collecting [early feedback](https://docs.google.com/document/d/1bCToFYC5GpUD5cUA-QepF9zDOaDGPNKyxH4FANzjrTY/edit) from people accessing the feature via Konami Code. 

```suggestion
```

Reply:
done!